code/modules/mob/living/blood.dm

```diff
@@ -46,28 +49,68 @@
 				to_chat(src, span_userdanger("Blood starts to tear your skin apart. You're going to burst!"))
 				investigate_log("has been gibbed by having too much blood.", INVESTIGATE_DEATHS)
 				inflate_gib()
+		// Too much blood
 		if(BLOOD_VOLUME_MAXIMUM to BLOOD_VOLUME_EXCESS)
 			if(SPT_PROB(5, seconds_per_tick))
 				to_chat(src, span_warning("You feel terribly bloated."))
+		// Slightly less blood than normal
+		if(BLOOD_VOLUME_SAFE to BLOOD_VOLUME_NORMAL)
+			if(SPT_PROB(1.5, seconds_per_tick))
+				set_eye_blur_if_lower(1 SECONDS * determined_mod)
+				to_chat(src, span_warning("You feel a little bit [word]."))
```

Comment:
I don't think there should be any effects between safe and normal.

This means you could be missing a single unit of blood and now you feel oooh dizzy ohh woozy